Easydict/NewApp/EasydictApp.swift

```diff
@@ -47,9 +42,13 @@ struct EasydictApp: App {
                 Label {
                     Text("Easydict")
                 } icon: {
-                    Image(menuBarImage)
+                    Image(menuBarIcon.rawValue)
                         .resizable()
+                    #if DEBUG
+                        .renderingMode(.original)
```

Comment:
I updated the menu bar image source, show original rendering mode in debug, which is blue.

<img width="270" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/6e171e3f-820b-493c-819b-89b3bcf979c0">

<img width="268" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/0c02a1ea-71f4-44d4-99c6-551b86752467">

Reply:
Looks very nice. Hopefully no users will request a feature to change the icon colour :)